docker-compose.yml

```diff
@@ -57,6 +57,13 @@ services:
     depends_on:
       - varnish
 
+  memcached:
+    build:
+      context: ./memcached/1.4/
+    image: quay.io/continuouspipe/memcached1.4:v1.0
```

Comment:
In both cases that means that we need to rethink the entire image versioning to be consistent then... I agree with this PR and we can proceed with every image name in another PR then :)